### PR TITLE
Enrollment date script

### PR DESCRIPTION
**Pull request by blootsvoets**
Description:
Ensure that the subject enrolment date matches its created date when the enrolment date variable is introduced.

### CODE REVIEW

File: src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml [hunk 1]
```diff
@@ -24,10 +24,26 @@
...
+    <changeSet id="add subject enrollment date" author="joris@thehyve.nl">
+        <sql>
+            UPDATE subject
+            SET subject.enrollment_date = (
+                SELECT MIN(r.timestamp)
+                FROM subject_aud a
+                    INNER JOIN _revisions_info r ON a.rev = r.id
+                WHERE a.id = subject.id
+                    AND a.revtype = 0
+                GROUP BY a.id
```
Comment (aykl) [inline src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml:44]: Isn't it redundant? Aggregate functions like `MIN` return one result even without `GROUP BY`.

File: src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml [hunk 2]
```diff
@@ -24,10 +24,26 @@
...
     </changeSet>
+
     <changeSet id="20211006170000-2" author="alexey@thehyve.nl">
         <addColumn tableName="subject_aud">
             <column name="enrollment_date" type="timestamp"/>
             <column name="enrollment_date_mod" type="BOOLEAN"/>
         </addColumn>
     </changeSet>
+
+    <changeSet id="add subject enrollment date" author="joris@thehyve.nl">
```
Comment (aykl) [inline src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml:35]: I like this id. It makes me wonder why don't we identify other changesets is the same readable manner.

**Comment by aykl** [comment]
LGTM.

File: src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml (hunk 2 above)
Comment (blootsvoets) [reply src/main/resources/config/liquibase/changelog/20211006170000_add_enrollment_date.xml:35]: I'm not sure either :-) I noticed this type of ID being used elsewhere and it's much more readable, also if looking through the liquidase change log table.

**Pull request merged**